storage/remote/wal_watcher_test.go

```diff
@@ -505,5 +505,5 @@ func TestCheckpointSeriesReset(t *testing.T) {
 	// If you modify the checkpoint and truncate segment #'s run the test to see how
 	// many series records you end up with and change the last Equals check accordingly
 	// or modify the Equals to Assert(len(wt.seriesLabels) < seriesCount*10)
-	testutil.Equals(t, 14, wt.checkNumLabels())
+	testutil.Equals(t, 13, wt.checkNumLabels())
```

Comment:
This change is due to the WAL compression causing a different number of segments.